boost-maven/boost-boms/booster-ee7-bom/pom.xml

```diff
@@ -1,64 +1,76 @@
+    <modelVersion>4.0.0</modelVersion>
 
-	<parent>
-		<groupId>boost</groupId>
-		<artifactId>boost-maven-parent</artifactId>
-		<version>0.1.3-SNAPSHOT</version>
-		<relativePath>../..</relativePath>
-	</parent>
+    <parent>
+        <groupId>boost</groupId>
+        <artifactId>boost-maven-parent</artifactId>
+        <version>0.1.3-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
 
-	<groupId>boost.boosters</groupId>
-	<artifactId>ee7-bom</artifactId>
-	<packaging>pom</packaging>
+    <groupId>boost.boosters</groupId>
+    <artifactId>ee7-bom</artifactId>
+    <packaging>pom</packaging>
 
-	<description>Booster EE7 Feature Bom</description>
+    <description>Booster EE7 Feature Bom</description>
 
-	<dependencyManagement>
-		<dependencies>
-			<dependency>
-				<groupId>boost.boosters</groupId>
-				<artifactId>boost-ee7-apis-bom</artifactId>
-				<version>0.1-SNAPSHOT</version>
-				<scope>import</scope>
-				<type>pom</type>
-			</dependency>
-			<dependency>
-				<groupId>boost.runtimes</groupId>
-				<artifactId>runtimes-bom</artifactId>
-				<version>0.1.3-SNAPSHOT</version>
-				<scope>import</scope>
-				<type>pom</type>
-			</dependency>
-			<dependency>
-				<groupId>boost.boosters</groupId>
-				<artifactId>jaxrs</artifactId>
-				<!-- Booster version distinct from the BOM, boost-maven-plugin version -->
-				<!-- Could also use 7.0, 8.0 for EE7, 8 -->
-				<!-- Probably will be 1.0 for initial GA -->
-				<version>0.1-SNAPSHOT</version>
-				<scope>provided</scope>
-			</dependency>
-			<dependency>
-				<groupId>boost.boosters</groupId>
-				<artifactId>jdbc</artifactId>
-				<version>0.1-SNAPSHOT</version>
-				<scope>provided</scope>
-			</dependency>
-			<dependency>
-				<groupId>boost.boosters</groupId>
-				<artifactId>jpa</artifactId>
-				<version>0.1-SNAPSHOT</version>
-				<scope>provided</scope>
-			</dependency>
-		</dependencies>
-	</dependencyManagement>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>boost.boosters</groupId>
+                <artifactId>boost-ee7-apis-bom</artifactId>
+                <version>0.1-SNAPSHOT</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+            <dependency>
+                <groupId>boost.runtimes</groupId>
+                <artifactId>runtimes-bom</artifactId>
+                <version>0.1.3-SNAPSHOT</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+            <dependency>
+                <groupId>boost.boosters</groupId>
+                <artifactId>jaxrs</artifactId>
+                <!-- Booster version distinct from the BOM, boost-maven-plugin 
+                    version -->
+                <!-- Could also use 7.0, 8.0 for EE7, 8 -->
+                <!-- Probably will be 1.0 for initial GA -->
+                <version>2.0-0-M1-SNAPSHOT</version>
```

Comment:
This looks a bit less readable than I thought...can we change this from `2.0-0-M1-SNAPSHOT` to `2.0-1.0-M1-SNAPSHOT`... so to start we'll align with the plugin version (which, pre-1.0, will be `1.0-M1-SNAPSHOT`  - though of course may diverge over time).

Reply:
Of course I meant to make similar changes for all boosters.

Reply:
Moving this comment to PR #337 and counting this "resolved".